### DISCUSSION TITLE
RETURNING paratmeter for INSERT

### DISCUSSION THREAD

**Discussion by Fogapod** [geldata/gel#1560] [Q&A] [answered]
I'm sure I saw some discussion about it last year, but can't find it now.
Is there an alternative for postgres's `INSERT INTO ... RETURNING ...`?

**Comment by elprans** [comment, marked as answer]
`INSERT` in EdgeDB is already "returning" (like every other expression), you don't need any special clauses for it, so:

```
WITH 
    Inserted := (INSERT ... )
SELECT
    Inserted { ... }
```
